include/fiction/io/print_layout.hpp

```diff
@@ -5,14 +5,21 @@
 #ifndef FICTION_PRINT_LAYOUT_HPP
 #define FICTION_PRINT_LAYOUT_HPP
 
+#include "fiction/technology/charge_distribution_surface.hpp"
 #include "fiction/traits.hpp"
 #include "fiction/types.hpp"
 
 #include <fmt/color.h>
 #include <fmt/format.h>
 
+#include <algorithm>
 #include <array>
+#include <cmath>
+#include <cstdint>
+#include <iostream>
+#include <limits>
 #include <string>
+#include <utility>
```

Comment:
Now that you removed the usage of `std::pair`, you do not need the `<utility>` header anymore.